src/v1/sources/cordial/transform.js

```diff
@@ -0,0 +1,44 @@
+const Message = require('../../../v0/sources/message');
+const { CommonUtils } = require('../../../util/common');
+const { generateUUID, isDefinedAndNotNull } = require('../../../v0/util');
+const { eventsMapping } = require('./config');
+
+const mapping = require('./mapping.json');
+
```

Comment:
Refere some cordial source doc for reference here

Reply:
Doc link: [](https://support.cordial.com/hc/en-us/articles/13584534464141-Webhooks)

Reply:
Add in the code itself